examples/hosts/app-integration/external-views/jest-puppeteer.config.js

```diff
@@ -10,10 +10,15 @@ module.exports = {
     launchTimeout:10000,
     usedPortAction: 'error'
   },
-  launch: {
-    args: ['--no-sandbox', '--disable-setuid-sandbox'], // https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#setting-up-chrome-linux-sandbox
-    dumpio: process.env.FLUID_TEST_VERBOSE !== undefined // output browser console to cmd line
-    // slowMo: 500, // slows down process for easier viewing
-    // headless: false, // run in the browser
-  },
+//   launch: {
```

Comment:
Seems maybe unintended to change the external-views sample?

Reply:
I changed back to external-views sample, but I uncommented "headless: false". When tests are running headless, the behavior seems to be sporadic (timeouts, some tests not passing, etc.) and I'm currently not sure why.

Reply:
That's probably fine, but I don't expect changes to files under /examples/hosts/app-integration/external-views/* in this PR.

Reply:
Oops yes you're right, was playing around in the wrong file.